src/modules/user/controller/userControllers.ts

```diff
@@ -0,0 +1,14 @@
+import { Request, Response } from "express";
+import httpStatus from "http-status";
+import userRepositories from "../repository/userRepositories";
+
+const updateUserStatus = async (req: Request, res: Response): Promise<void> => {
+  try {
+    const userId: number = Number(req.params.id);
+    const data = await userRepositories.UpdateUserByAttributes(userId, req.body.status);
```

Comment:
We already have the `UpdateUserByAttributes` function into `e-commerce-ninjas-bn\src\modules\auth\repository\authRepositories.ts` which is doing the same job.